src/app/clients/create-address/create-address.component.html

```diff
@@ -1,71 +1,71 @@
-                  <div>
-
-                    <mat-option *ngFor="let item of clientAddress?.addressTypeIdOptions" name="option" [value]="item?.id">{{item?.name}}</mat-option>
-                  </div>
-                </mat-select>
-              </mat-form-field>
-
-              <mat-form-field class="example-full-width">
-                <input matInput name="street" placeholder="Street" ngModel>
-              </mat-form-field>
-              <mat-form-field class="example-full-width">
-                <input matInput name="addressLine1" placeholder="Address Line 1" ngModel>
-              </mat-form-field>
-
-
-              <mat-form-field class="example-full-width">
-                <input matInput name="addressLine2" placeholder="Address Line 2" ngModel>
-              </mat-form-field>
-              <mat-form-field class="example-full-width">
-                <input matInput name="addressLine3" placeholder="Address Line 3" ngModel>
-              </mat-form-field>
-              <mat-form-field class="example-full-width">
-                <input matInput name="city" placeholder="City" ngModel>
-              </mat-form-field>
-              <mat-form-field class="example-full-width">
-                <mat-select placeholder="State" [(ngModel)]="addressStateId" [ngModelOptions]="{standalone: true}" ngDefaultControl>
-                  <div>
-
-                    <mat-option *ngFor="let item of clientAddress?.stateProvinceIdOptions" name="option" [value]="item?.id">{{item?.name}}</mat-option>
-                  </div>
-                </mat-select>
-              </mat-form-field>
-              <mat-form-field class="example-full-width">
-                <mat-select placeholder="Country" [(ngModel)]="addressCountryId" [ngModelOptions]="{standalone: true}" ngDefaultControl>
-                  <div>
-
-                    <mat-option *ngFor="let item of clientAddress?.countryIdOptions" name="option" [value]="item?.id">{{item?.name}}</mat-option>
-                  </div>
-                </mat-select>
-              </mat-form-field>
-              <mat-form-field class="example-full-width">
-                <input matInput name="postalCode" placeholder="Postal Code" ngModel>
-              </mat-form-field>
-
-              <button mat-raised-button>Create</button>
-
-            </form>
-
-          </mat-card-content>
-        </mat-card>
-
-
-      </div>
-
-
-    </aw-wizard-step>
-  </aw-wizard>
+<div class="container">
+  <mat-card>
+    <form [formGroup]="newAddressForm" (ngSubmit)="submit()">
+      <mat-card-content>
+        <div fxLayout="row wrap" fxLayoutGap="2%" fxLayout.lt-md="column">
+          <mat-form-field fxFlex="48%">
+            <mat-label>Address Type</mat-label>
+            <mat-select required formControlName="addressTypeID">
+              <mat-option *ngFor="let item of clientAddress?.addressTypeIdOptions" [value]="item?.id">
```

Comment:
Please give a relevant name instead of `item`. What is the need of adding `?`? Load the data before routing to this component using `resolvers`  and remove the `?`.